src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  if (operation === 'X') {
+    res = Big(numberOne) * Big(numberTwo);
```

Comment:
- [ ] Mathematical operations should be handled by Big.js library. Kindly check `plus, minus, times, div, ..` methods from Big.js [docs](https://mikemcl.github.io/big.js/#prototype-methods)